### DISCUSSION TITLE
Option to go back to page x after some time

### DISCUSSION THREAD

**Discussion by AntoineLemaire** [streamdeck-linux-gui/streamdeck-linux-gui#75] [Ideas / Feature Requests]
I see lot of users (including me), using pages like some folders, each folder has a "back" button to return to main page.

I could be cool to have a option to automaticly return to page x after some defined inactivity time

**Comment by Danie10** [comment]
It will depend much upon what one uses that page for - some activities may be regular button presses, others may be longer periods of inactivity. I have a back button on mine, as I also drop three pages deep for some activities, and I use the back button to back up just to the previous page only (not to start/pg 1).

But yes an optional tick for that could work. Or else just a value field with default 0 for never, or >0 for the specified time. Would mean tracking the active page for that.
